### PR TITLE
Favorites

### PR DESCRIPTION
**Pull request by megabytos**

### CODE REVIEW

File: src/js/exercise-modal.js [hunk 1]
```diff
@@ -4,28 +4,42 @@ const modal = document.getElementById('exerciseModal');
 const closeButton = document.getElementById('closeButton');
 const addFavoritesButton = document.getElementById('addFavoritesButton');
 const removeFavoritesButton = document.getElementById('removeFavoritesButton');
+const FAV_KEY = 'favorites';
```
Comment (saniola) [inline src/js/exercise-modal.js]: let's reuse from constants

File: src/js/exercise-modal.js [hunk 2]
```diff
@@ -4,28 +4,42 @@ const modal = document.getElementById('exerciseModal');
...
   let exerciseData;
   try {
     exerciseData = await fetchExercise(exerciseId);
     renderModal(exerciseData);
     addCloseButtonListener();
-    addFavoritesListener();
-  } catch (error) {}
+    initFavoritesButton();  
+  } catch (error) {
+    console.error('Error fetching exercise data:', error);
```
Comment (saniola) [inline src/js/exercise-modal.js]: Please use iziTost here

File: src/js/favorites.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+const filter = 'Muscles';
+let page = 1;
+let catValue = '';
+
+document.addEventListener('DOMContentLoaded', async () => {
+ 
+  fetchFavorites();
+
+
```
Comment (saniola) [inline src/js/favorites.js]: indents

**Review by saniola — APPROVED** [review]

File: src/js/fetch-favorites.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import { constants } from './constants';
+import { createExerciseMarkup } from './exercise-card-markup';
+import { createPagination } from './create-pagination';
+
+export async function fetchFavorites(params) {
+  const param = params ? params : { page: 1 };
+  const { data } = await axios({
```
Comment (Netaly79) [inline src/js/fetch-favorites.js:7]: Could you add try-catch-finally block (check another fetch file)

**Pull request merged**